### ISSUE TITLE
Search case insensitive

### ISSUE THREAD

**Issue by Jeddix** [jacquesh/foo_openlyrics#59]
Could you please make the search case insensitive? Some artists make their song titles or albums upper case but in my library they are saved in normal case. Open Lyrics then uses the first search result that is found with normal case, even though it's unsynced.

![grafik](https://user-images.githubusercontent.com/2885473/133922627-2558c791-8155-4670-8649-d61a0bc136ff.png)

**Comment by jacquesh** [comment]
Good idea, I'm pretty sure I've run into this myself in the past actually. Will do.
